src/index.ts

```diff
@@ -33,7 +33,6 @@ import {
 } from './configuration-store/configuration-store';
 import { HybridConfigurationStore } from './configuration-store/hybrid.store';
 import { MemoryStore, MemoryOnlyConfigurationStore } from './configuration-store/memory.store';
-import { ConfigurationWireHelper } from './configuration-wire/configuration-wire-helper';
```

Comment:
I think this makes sense, although it may also make sense to export it from the node-server SDK to allow customers to utilize it without having to install both the common and node server SDKs